spec/player_spec.rb

```diff
@@ -90,6 +92,36 @@
       end
     end
 
+    describe "#player_tiles" do
+      it "returns a collection of tiles that the player can play" do
+        expect(@player.player_tiles.length).to eq(0)
+        expect(@player.player_tiles).to be_an(Array)
+      end
+    end
+
+    describe "#draw_tiles(tile_bag)" do
+      before :all do
+        @playerJ = Scrabble::Player.new("J")
+        @tilebag2 = Scrabble::TileBag.new
```

Comment:
Is there a reason that we are creating a new `TileBag` instance here and not using the `@tilebag` variable created in the `before` block from L9?